### PR TITLE
tests: coverage increment

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: src/invenio-search-js/templates/invenioSearchResultsPagination.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
+{{ hello }}
```
Comment (jirikuncar) [inline src/invenio-search-js/templates/invenioSearchResultsPagination.html]: is it debug or necessary part?

Comment (drjova) [inline src/invenio-search-js/templates/invenioSearchResultsPagination.html]: :see_no_evil: removed thanks!

**Comment by jirikuncar** [comment]
rebase ping @drjova :rocket:

**Comment by drjova** [comment]
@jirikuncar done :dancer:

**Pull request merged**